r2/r2/templates/linkinfobar.html

```diff
@@ -39,7 +39,7 @@
       &#32;
     </span>
     <time datetime="${html_datetime(thing.a._date)}">
-      ${thing.a._date.strftime(thing.datefmt)}
+      ${thing.a._date.strftime(_('%d %b %Y'))}
```

Comment:
something from babel might be more appropriate http://babel.pocoo.org/docs/api/dates/